airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/integrations/destination/dest_state_lifecycle_manager/DestStreamStateLifecycleManager.java

```diff
@@ -50,7 +49,8 @@ public void addState(final AirbyteMessage message) {
     Preconditions.checkArgument(message.getState().getType() == AirbyteStateType.STREAM);
     final StreamDescriptor originalStreamId = message.getState().getStream().getStreamDescriptor();
     final StreamDescriptor actualStreamId;
-    if (StringUtils.isNullOrEmpty(originalStreamId.getNamespace())) {
+    final String namespace = originalStreamId.getNamespace();
+    if (namespace == null || namespace.isEmpty()) {
```

Comment:
:joy:

how much build time did we lose to save the half-second of time writing this code?